src/components/Tooltip/index.js

```diff
@@ -60,8 +61,15 @@ export const StyledTooltip = styled.div`
     }
 `
 
-const Tooltip = props => (
-    <TooltipTrigger
+const Tooltip = props => {
+    const tooltipContent = isString(props.tipText)
```

Comment:
lets get rid of this strange code, we can pass `<p>` tags in the children if needed

Reply:
@SamDavies The problem is that the current API uses children for the content of the trigger and `tipText` for the content so if we can change this API, it'll need changed across the whole app as the changes will be breaking.  I'll try and figure out a better API and then add a complimentary PR in the app once it's approved.

Reply:
maybe something like:
```js
<Tooltip>
  <TooltipContent>
    // ... whatever elements you want in here
  </TooltipContent>
  <TooltipTrigger>
    // ... button or something
  </TooltipTrigger>
</Tooltip>
```

Reply:
perhaps:
```
<Tooltip
    renderTooltip={() => <div>...</div>}
>
  <TooltipTrigger>
    // ... button or something
  </TooltipTrigger>
</Tooltip>
```